Jobs.Fetcher.TikTok/Jobs/PostsQuery.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Serilog.Core;
+using DataLakeModels;
+using Jobs.Fetcher.TikTok.Helpers;
+
+namespace Jobs.Fetcher.TikTok {
+
+    public class PostsQuery : AbstractTikTokFetcher {
+
+        public PostsQuery(List<string> userIds): base(userIds) {}
+
+        public override List<string> Dependencies() {
+            return new List<string>() {};
+        }
+
+        public override void RunBody(string username) {
+            using (var dbContext = new DataLakeTikTokContext()) {
+                //Create a second loop for each TikTok client. Check with Erik and Victor if the Get option will receive an ID or username
```

Comment:
The interface for the scraper has been changed. Now we are using configuration files with JSON format (`jobs` directory tries to match the structure of `Credentials` directory in Andromeda), and the parameter read is the username

Reply:
I believe I had already fixed this, but I left this comment from long ago